jobs/build/build-sync/build.groovy

```diff
@@ -90,6 +90,49 @@ ${excludeArchesParam}
 ${dryRunParams}
 """
     }
+
+    if (params.ASSEMBLY == "stream" && !params.DOOZER_DATA_GITREF) {
+        slackChannel = slacklib.to(params.VERSION)
+        slackChannel.task("Synchronizing ART CoreOS with CI for ${params.VERSION}") {
+            buildlib.oc("${logLevel} --kubeconfig ${buildlib.ciKubeconfig} registry login")
+            def (major, minor) = commonlib.extractMajorMinorVersionNumbers(params.BUILD_VERSION)
+            // Starting with 4.12, ART is responsible for populating the CI imagestream (-n ocp is/4.12) with
+            // references to the latest machine-os-content, rhel-coreos-8, rhel-coreos-8-extensions (and
+            // potentially more with rhel9). If this is failing, it must be treated as a priority since
+            // CI will begin falling being nightly CoreOS content.
+            if ( major > 4 || minor >= 12 ) {
```

Comment:
Shouldn't this be `major >= 4`?

Reply:
This looks like the only major issue with this PR.

Reply:
The >= 12 captures 4.12 and beyond.

Reply:
Ah right sorry yep